docs/source/references.bib

```diff
@@ -160,6 +160,17 @@ @TechReport{IOGP2018
   Url                      = {https://www.iogp.org/bookstore/product/coordinate-conversions-and-transformation-including-formulas/}
 }
 
+@TechReport{ISO19111,
+type = {Standard},
+key = {ISO 19111:2019},
+month = Jan,
+year = {2019},
+title = {{Geographic information -- Referencing by coordinates}},
+volume = {2019},
+address = {Geneva, CH},
+institution = {International Organization for Standardization}
```

Comment:
instead of people hitting a pay wall, we can direct them to
http://docs.opengeospatial.org/as/18-005r4/18-005r4.html
which is the same as ISO 191111:2019

Travis OSX complains about 'missing author in ISO19111'

Reply:
I've added the OGC URL to the bibtex entry. And added "ISO" as author.